cinder/scheduler/rpcapi.py

```diff
@@ -245,3 +245,8 @@ def set_log_levels(self, context, service, log_request):
     def get_log_levels(self, context, service, log_request):
         cctxt = self._get_cctxt(server=service.host, version='3.7')
         return cctxt.call(context, 'get_log_levels', log_request=log_request)
+
+    def find_backend_for_connector(self, context, connector, request_spec):
+        cctxt = self._get_cctxt()
```

Comment:
I think a version will be required once this will be submitted upstream, but I tried to avoid doing that now, since I wasn't sure what version to put in here.
Maybe, for now, we could set a minor version, like `3.7.1`, or something else, to not interfere with what's upstream.